<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base1" output="screen" clear_params="true" args="/map:=projected_map /odom:=Odometry">
        <rosparam file="$(find loong_planner)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find loong_planner)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find loong_planner)/param/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find loong_planner)/param/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find loong_planner)/param/loong_planner_params.yaml" command="load" />

        <param name="base_global_planner" value="global_planner/GlobalPlanner" /> 
        <param name="base_local_planner" value="loong_planner/LoongPlanner" />
    </node>
    <node pkg="sentry_nav" type="trans_tf_2d" name="trans_tF_2d" />
    <!-- <node pkg="sentry_nav" type="trans_tf_2d_odom" name="trans_tF_2d_odom" />
    <node pkg="sentry_nav" type="trans_odom_2d" name="trans_odom_2d" /> -->
	<include file="$(find velocity_smoother_ema)/launch/velocity_smoother_ema.launch">
	</include>
</launch>
